client-src/runtar.c

```diff
@@ -192,8 +192,9 @@ main(
 		g_str_has_prefix(argv[i],"--exclude-from") ||
 		g_str_has_prefix(argv[i],"--files-from")) {
 		good_option++;
-	    } else if (argv[i][0] != '-') {
```

Comment:
Would it work to omit this change and leave the one I suggested above? This would not treat `-` as a special case.

Reply:
yes ! 👍🏻